upf/upf_pfcp.c

```diff
@@ -1020,6 +1021,17 @@ pfcp_disable_session (upf_session_t * sx)
   return 0;
 }
 
+void
+upf_delete_nat_binding (upf_session_t * sx)
+{
+  upf_nat_addr_t *ap = sx->nat_addr;
+  if (ap->used_blocks > 0)
+    ap->used_blocks -= 1;
+  upf_nat_del_binding =
```

Comment:
wouldn't it make more sense to resolve the symbol only once during initialization?

Reply:
Yes, it would. But it turned out that pointer to a function loosing reference.
In VPP code is general thing to resolve symbol right before calling the function.